hack/boilerplate/boilerplate.go.txt

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021 Google LLC All Rights Reserved.
+// Copyright 2021 ko Build Authors All Rights Reserved.
```

Comment:
Sorry to keep moving the target. I think this should be `Copyright 20XX ko Authors All Rights Reserved`.

Reply:
no worries!

Reply:
cannot change that :/ the job check fails then

```
Error: [Go headers] reported by reviewdog 🐶
  missing boilerplate:
  
  Raw Output:
  main.go:1: missing boilerplate:
  // Copyright 20XX ko Build Authors All Rights Reserved.
  //
  // Licensed under the Apache License
```